src/components/CanvasAnnotations.js

```diff
@@ -0,0 +1,115 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import List from '@material-ui/core/List';
+import ListItem from '@material-ui/core/ListItem';
+import ListItemText from '@material-ui/core/ListItemText';
+import Typography from '@material-ui/core/Typography';
+import { SanitizedHtml } from './SanitizedHtml';
+
+/**
+ * CanvasAnnotations ~
+*/
+export class CanvasAnnotations extends Component {
+  /**
+   * constructor -
+   */
+  constructor(props) {
+    super(props);
+
+    this.handleClick = this.handleClick.bind(this);
+    this.handleMouseEnter = this.handleMouseEnter.bind(this);
+    this.handleMouseLeave = this.handleMouseLeave.bind(this);
+  }
+
+  /**
+   * Handle click event of an annotation.
+  */
+  handleClick(event, annotation) {
+    const {
+      deselectAnnotation, selectAnnotation, selectedAnnotationIds, windowId,
+    } = this.props;
+
+    if (selectedAnnotationIds.includes(annotation.id)) {
+      deselectAnnotation(windowId, annotation.targetId, annotation.id);
+    } else {
+      selectAnnotation(windowId, annotation.targetId, annotation.id);
+    }
+  }
+
+  /** */
+  handleMouseEnter(annotation) {
+    const { allAnnotationsAreHighlighted, highlightAnnotation, windowId } = this.props;
+    if (allAnnotationsAreHighlighted) return;
+
+    highlightAnnotation(windowId, annotation.id);
+  }
+
+  /** */
+  handleMouseLeave() {
+    const { allAnnotationsAreHighlighted, highlightAnnotation, windowId } = this.props;
+    if (allAnnotationsAreHighlighted) return;
+
+    highlightAnnotation(windowId, null);
+  }
+
+  /**
+   * Returns the rendered component
+  */
+  render() {
+    const {
+      annotations, classes, index, label, selectedAnnotationIds, t, totalSize,
+    } = this.props;
+    if (annotations.length === 0) return <></>;
```

Comment:
I don't _think_ this was specified, but I'm not rendering anything (so no overline w/ the canvas label) when there are no annotations.